Sniffs/Commenting/ContractDocBlockSniff.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class ContractDocBlockSniff implements Sniff
+{
+    use DocBlockTrait;
+
+    /**
+     * @return mixed[]|void
+     */
+    public function register()
+    {
+        return [
+            T_INTERFACE,
+        ];
+    }
+
+    /**
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile
+     * @param $stackPtr
+     *
+     * @return int|void
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $contractPtr = $phpcsFile->findPrevious([T_INTERFACE], $stackPtr);
```

Comment:
Can you use ```$this->register()``` instead of ```[T_INTERFACE]``` here ?
Or move add method like that, and use it in this class.
```php
protected function getTargets()
{
        return [
            T_INTERFACE,
        ];
}
```

Reply:
It needs to find an exact pointer. It is already used in `register()` method. Just another purpose.